src/codewhisperer/commands/onInlineAcceptance.ts

```diff
@@ -73,7 +74,10 @@ export async function onInlineAcceptance(
 
     if (acceptanceEntry.editor) {
         await sleep(CodeWhispererConstants.vsCodeCursorUpdateDelay)
-        const languageContext = runtimeLanguageContext.getLanguageContext(acceptanceEntry.editor.document.languageId)
+        const languageContext = runtimeLanguageContext.getLanguageContext(
+            acceptanceEntry.editor.document.languageId,
+            getFileExtension(acceptanceEntry.editor.document)
+        )
```

Comment:
Looks like all of these calls could benefit if `getLanguageContext` accepted a document (could be an optional overload to support both ways)
```suggestion
        const languageContext = runtimeLanguageContext.getLanguageContext(editor.document)
```

Reply:
responded below.

Reply:
Can't that be solved with an overload?

Reply:
`getLanguageContext` needs more refactoring along with other functions `mapToRuntimeLanguage` to do this.